cmd/client/txgen/main.go

```diff
@@ -98,7 +98,7 @@ func setUpTXGen() *node.Node {
 		fmt.Fprintf(os.Stderr, "Error :%v \n", err)
 		os.Exit(1)
 	}
-	consensusObj, err := consensus.New(myhost, uint32(shardID), p2p.Peer{}, nil)
+	consensusObj, err := consensus.NewOneVotePerValidator(myhost, uint32(shardID), p2p.Peer{}, nil)
```

Comment:
One validator can have different voting power, instead of each one has equal power, we can add a field votingPower in consensus object which is calculated from their stake

Reply:
yes, see the other constructor function which is for staking based voting.

Reply:
https://github.com/harmony-one/harmony/pull/1717/files#diff-3a5c8ea459d421751fb70343be214598R251